tests/standalone/six.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import
+import six
+import sys
+from logging import handlers
+
+
+PY3 = sys.version_info[0] == 3
+if PY3:
+ string_types = str,
+ integer_types = int,
+ class_types = type
+
+
+def dispatch_types(value):
+   
+    if isinstance(value, six.integer_types):
```

Comment:
This is not a big coverage of the six package, is it?

Reply:
Okay, I think I should expand the usage of Six specifically.